ci/install-aws.inc.sh

```diff
@@ -11,9 +11,21 @@ if [[ "$OS" = "linux" ]] && [[ "${FORCE_BREW:-}" != "true" ]]; then
     pip3 install --upgrade awscli
     echo_done
 else
-    echo_do "brew: Installing AWS utils..."
-    brew_install awscli
-    echo_done
+    if [[ ${GITHUB_ACTIONS:=} == "true" ]]; then
+        # https://github.com/actions/virtual-environments/commit/0fa2247a898625b8020d8dfb1dc4a54ddb2b256c
+        # Github Actions CI already installs awscli via pkg,
+        # which will make 'brew install awscli' fail when linking.
+        # See https://github.com/rokmoln/support-firecloud/runs/1291359454
+        pkgutil --pkgs | grep -q "^com\.amazon\.aws\.cli2" && HAS_AWSCLI_PKG=true || HAS_AWSCLI_PKG=false
```

Comment:
Subjective and minor: perhaps we don't need to set `HAS_AWSCLI_PKG` to false since it will not be set to anything, ie undefined, if we are not on GITHUB_ACTIONS. It just seems easier to only deal with `true` or undefined instead of `true`, `false` and undefined.

At least this made be have to read the code a couple of extra times to understand what was going on.